# -*- coding: utf-8 -*-
# Generated by Django 1.11.20 on 2019-03-27 17:35
from __future__ import unicode_literals

import sys

from django.db import migrations


def insert_sync_failed_status_fixtures(apps, schema_editor):
    database = 'default'
    try:
        i = sys.argv.index('--database')
        database = sys.argv[i + 1]
    except ValueError:
        pass

    AnnotationJobStatus = apps.get_model('backlog', 'AnnotationJobStatus')
    try:
        AnnotationJobStatus(description='SYNC-FAILED').save(using=database)
    except Exception as e:
        print(e)


class Migration(migrations.Migration):
    dependencies = [
        ('backlog', '0054_auto_20190225_1632'),
    ]

    operations = [
        migrations.RunPython(insert_sync_failed_status_fixtures)
    ]
